rooms.js

```diff
@@ -865,8 +867,7 @@ class GlobalRoom extends BasicRoom {
 	startLockdown(err, slow = false) {
 		if (this.lockdown && err) return;
 		let devRoom = Rooms('development');
-		// @ts-ignore
-		const stack = (err ? Chat.escapeHTML(err.stack).split(`\n`).slice(0, 2).join(`<br />`) : ``);
+		const stack = (err.stack ? Chat.escapeHTML(err.stack).split(`\n`).slice(0, 2).join(`<br />`) : ``);
```

Comment:
This will crash if `err` is `null`, which, contrary to the function signature, it might be.

Reply:
(Specifically, it will be if you use `/lockdown`.)